pkg/operation/botanist/component/nginxingress/nginxingress.go

```diff
@@ -579,6 +583,9 @@ func (n *nginxIngress) computeResourcesData() (map[string][]byte, error) {
 	// Skipped until https://github.com/kubernetes/ingress-nginx/issues/8640 is resolved
 	// and special seccomp profile is implemented for the nginx-ingress
 	deploymentController.Spec.Template.Labels[resourcesv1alpha1.SeccompProfileSkip] = "true"
+	if k8sVersionGreaterEqual119 {
```

Comment:
Nit: Gardener supports only Seed clusters with K8s >= 1.20.

https://github.com/gardener/gardener/blob/98ce217fd6bc916e9f08fb0d63d6480ab13ea7ab/pkg/gardenlet/controller/seed/seed/reconciler_reconcile.go#L199-L211

We can omit this version check and assume that the Seed's K8s version is >= 1.20.

Reply:
Yes, you are right! I will make the change